content/applications/sales/sales/invoicing/proforma.rst

```diff
@@ -7,23 +7,24 @@ notes the kind and quantity of goods, their value, and other important informati
 and transportation charges.
 
 Pro-forma invoices are commonly used as preliminary invoices with a quotation. They are also used
-for customs purposes during importation. They differ from a normal invoice in that they are not a
+during importation for customs purposes. They differ from a normal invoice, in that they are *not* a
```

Comment:
```suggestion
during importation, for customs purposes. They differ from a normal invoice, in that they are *not* a
```

Reply:
I don't believe a comma is necessary in this instance, as its a fairly short, straightforward sentence. So, I think I'll leave it as-is. However, if you believe I am incorrect, just let me know, and I'll make the change right away. Thanks!